src/test/java/rest/demo/controller/ToDoControllerTest.java

```diff
@@ -56,41 +54,75 @@ void getToDos_Accepted() {
     }
 
     @Test
-    @DisplayName("when ToDos do not exist in repository, then we get NOT_FOUND")
+    @DisplayName("When no ToDos, Then NOT_FOUND")

```

Comment:
when no ToDo retrieved/received, return NOT_FOUND

Reply:
maybe same for other tests

Reply:
done